src/stream/src/common/log_store_impl/kv_log_store/reader.rs

```diff
@@ -477,6 +514,7 @@ impl<S: StateStoreRead> LogReader for KvLogStoreReader<S> {
                 ));
             }
             KvLogStoreReaderFutureState::Empty => {}
+            KvLogStoreReaderFutureState::Reset(_) => {}
```

Comment:
Should be `unreachable`, because we must call `start_from` for a `Reset` reader.